crates/pevm/tests/mainnet.rs

```diff
@@ -57,3 +57,44 @@ fn mainnet_blocks_from_disk() {
         }
     });
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(all(feature = "rpc-storage", feature = "optimism"))]
+async fn optimism_mainnet_blocks_from_rpc() {
+    use alloy_provider::{Provider, ProviderBuilder};
+    use alloy_rpc_types_eth::{BlockId, Block};
+    use pevm::chain::{PevmChain, PevmOptimism};
+    use op_alloy_rpc_types::Transaction as OpTransaction;
+    use std::borrow::Cow;
+    use serde_json::json;
+
+    let rpc_url = match std::env::var("OPTIMISM_RPC_URL") {
+        Ok(value) if !value.is_empty() => value.parse().unwrap(),
+        _ => reqwest::Url::parse("https://mainnet.optimism.io").unwrap(),
```

Comment:
This is nit, but default OP rpc doesn't work for me :(

Could you switch to this one:
```suggestion
        _ => reqwest::Url::parse("https://rpc.ankr.com/optimism").unwrap(),
```